tests/test_postgresql.py

```diff
@@ -346,7 +346,7 @@ def test_remove_data_directory(self):
         self.p.remove_data_directory()
 
     @patch('subprocess.check_output', MagicMock(return_value=0, side_effect=pg_controldata_string))
-    @patch('subprocess.check_output', side_effect=subprocess.CalledProcessError)
+    @patch('subprocess.check_output', side_effect=subprocess.CalledProcessError(1, ''))
```

Comment:
This test is wrong on more than one layer (and your fix doesn't do much). Just leave it out the pull request, I will fix it with a separate one.